src/__test_resources__/dashboardsStructure.ts

```diff
@@ -0,0 +1,1994 @@
+import { ContentRecord } from "@activeviam/activeui-sdk-5.0";
+
+/**
+ * Example of the dashboard's structure part of a content server's ui folder.
+ * Can be found on the content server under /ui/dashboards/structure. 
+ */
+export const dashboardsStructure: ContentRecord = {
```

Comment:
I divided the length of this file by 2, but it can be shortened again if needed.